ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -40,6 +40,7 @@ StatusDialog {
     property alias preSelectedRecipientType: recipientInputLoader.selectedRecipientType
 
     property string preDefinedAmountToSend
+    property int preSelectedChainId: 0
```

Comment:
Do we have an enum to represent the chains so we can asign everywhere the default one instead of a `0` value?